ctapipe/containers.py

```diff
@@ -971,10 +971,29 @@ class MuonParametersContainer(Container):
     )
     intensity_ratio = Field(nan, "Intensity ratio of pixels in the ring to all pixels")
     mean_squared_error = Field(
-        nan, "MSE of the deviation of all pixels after cleaning from the ring fit"
+        nan * u.deg**2,
+        "MSE of the deviation of all pixels after cleaning from the ring fit",
     )
 
 
+class MuonCameraContainer(Container):
+    """
+    Container for muon analysis
+    """
+
+    ring = Field(default_factory=MuonRingContainer, description="muon ring fit")
+    parameters = Field(
+        default_factory=MuonParametersContainer, description="muon parameters"
+    )
+    efficiency = Field(
+        default_factory=MuonEfficiencyContainer, description="muon efficiency"
+    )
+
+
+class MuonContainer(Container):
+    tel = Field(default_factory=partial(Map, MuonCameraContainer))
```

Comment:
Wouldn't it make sense to call these `TelescopeContainer` instead of CameraContainer since we want to discard the CameraFrame altogether?  I am not into muon analysis, so maybe this might be the one instance where Camera is more sensible?

Reply:
Yes, I'd go so far to rename most of the containers from camera to telescope. After all, we store them in .tel not .cam

Reply:
That's where I was heading with this comment :+1: